docker/Dockerfile

```diff
@@ -28,15 +28,13 @@ RUN python /opt/pip.py
 RUN pip install ccm
 
 # Start Hawkular Metrics Cassandra Cluster
-CMD ccm create -v 3.0.12 -n 1 -s hawkular
+RUN ccm create -v 3.0.12 -n 1 -s hawkular
 
 # Clone the repository
 RUN git clone https://github.com/hawkular/hawkular-metrics.git /opt/hawkular-metrics
 
 # http, https, management, debug ports, Cassandra
 EXPOSE 8080 9990 8787 9042
 
-CMD cd /opt/hawkular-metrics/ && mvn clean install -DskipTests
-
 # Starting Hawkular Metrics
-CMD /opt/hawkular-metrics/integration-tests/rest-tests-jaxrs/target/wildfly-run/wildfly-10.0.0.Final/bin/standalone.sh
+CMD   cd /opt/hawkular-metrics/ && mvn clean install -DskipTests && /opt/hawkular-metrics/integration-tests/rest-tests-jaxrs/target/wildfly-run/wildfly-10.0.0.Final/bin/standalone.sh
```

Comment:
Not to mention the ccm creation, which will download Cassandra, install it (or compile it) and then initialize it..

This container will probably take ~10 minutes to start. Isn't this kinda defeating the whole idea? Also, the hawkular-metrics should probably be started from dist/standalone/wildfly-standalone/target/wildfly/wildfly-10.0.0.Final/bin/standalone.sh